glide.lock

```diff
@@ -109,7 +113,7 @@ imports:
   - mixer/adapter/model/v1beta1
   - policy/v1beta1
 - name: istio.io/istio
-  version: 42773aacced474d97159902d20579a25b1f98106
+  version: d639408fded355fb906ef2a1f9e8ffddc24c3d64
```

Comment:
Do we need to update Istio? Also, please commit the `github.com/mackerelio/go-osstat` package in the `vendor` directory for CI to pass (I'd do that in a separate PR to keep this PR succinct).

Reply:
It seems that `glide` cannot selectively install/update dependencies. I suggest that you undo this line manually and the istio package update.

We need the vendor directory committed as we'd prefer not to rely on the provider uptime for building the adapter.

Reply:
done